packages/react-day-picker/src/contexts/Focus/FocusContext.tsx

```diff
@@ -83,75 +86,32 @@ export function FocusProvider(props: { children: ReactNode }): JSX.Element {
     setFocusedDay(date);
   };
 
-  const focusDayBefore = () => {
+  const moveFocus = (addFn: typeof addDays, after: boolean) => {
```

Comment:
I like this change - I'd better move it in an external function tho... I will update the code when merged.